Formula/pdnsrec.rb

```diff
@@ -12,40 +12,49 @@ class Pdnsrec < Formula
     sha256 "a9faf9edf0e71de5e9ec9fa70d27811f353a8451b41746208fbdc0d592aa5910" => :mountain_lion
   end
 
+  head do
+    url "https://github.com/powerdns/pdns.git"
+
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+    depends_on "libtool"  => :build
+    depends_on "pandoc"   => :build
+    depends_on "ragel"    => :build
+  end
+
   depends_on :macos => :lion
   depends_on "boost"
   depends_on "lua" => :optional
 
+  depends_on "pkg-config" => :build if build.with? "lua"
+
   def install
-    # Set overrides using environment variables
-    ENV["DESTDIR"] = "#{prefix}"
-    ENV["OPTFLAGS"] = "-O0"
-    ENV.O0
+    args = %W[
+      --prefix=#{prefix}
+      --sysconfdir=#{prefix}/etc/powerdns
+      --with-openssl=#{Formula["openssl"].opt_prefix}
+    ]
 
     # Include Lua if requested
     if build.with? "lua"
-      ENV["LUA"] = "1"
-      ENV["LUA_CPPFLAGS_CONFIG"] = "-I#{Formula["lua"].opt_include}"
-      ENV["LUA_LIBS_CONFIG"] = "-llua"
+      args << "--with-lua"
     end
 
-    # Adjust hard coded paths in Makefile
-    inreplace "Makefile.in", "/usr/sbin/", "#{sbin}/"
-    inreplace "Makefile.in", "/usr/bin/", "#{bin}/"
-    inreplace "Makefile.in", "/etc/powerdns/", "#{etc}/powerdns/"
-    inreplace "Makefile.in", "/var/run/", "#{var}/run/"
+    inreplace "pdns_recursor.cc", "SYSCONFDIR", "\"#{etc}/powerdns/\""
+    inreplace "rec_control.cc", "SYSCONFDIR", "\"#{etc}/powerdns/\""
+
+    # Bootstrap?
+    Dir.chdir "pdns/recursordist" if build.head?
```

Comment:
Yes, I am. pdns_server, pdns-recursor and dnsdist are distinct products that live in one git tree because they share a lot of code. The non-HEAD versions are distributed as tarballs that are each individually complete (and thus overlap a bit), but when building from git, getting dnsdist or the recursor requires a chdir first.